ofcourse-templates/master.html

```diff
@@ -54,19 +59,25 @@
         <div class="col-md-3">
           <div class="well">
             <ul class="list-unstyled">
-              <li><span class="glyphicon glyphicon-calendar"></span> ${course['start']} &mdash; ${course['end']}</li>
-              <li><span class="glyphicon glyphicon-bell"></span> ${course['times']}</li>
-              <li><span class="glyphicon glyphicon-shopping-cart"></span> ${course['course']}</li>
-              <li><span class="glyphicon glyphicon-map-marker"></span> ${course['location']}</li>
+              <li><span class="glyphicon glyphicon-calendar"></span> {{course['start']}} &mdash; {{course['end']}}</li>
+              <li><span class="glyphicon glyphicon-bell"></span> {{course['times']}}</li>
```

Comment:
I think in dicts you can use "dict_name.foo" to reference properties, which would make this look way cleaner.